Added support for quotes around transcript name output.

apuignav opened the pull request:
Apparently some LaTeX versions (or maybe when the file name contains
spaces?) add quotes around the file name in the "Transcript written on
..." string that is used for getting the job name. Modified the regex to
allow for this possibility. ""

aclements commented on latexrun:
```diff
@@ -836,15 +836,15 @@ class LaTeX(Task):
         """
         jobname = outname = None
-        for m in re.finditer(r'^Transcript written on (.*)\.log\.$', stdout,
+        for m in re.finditer(r'^Transcript written on (")?(.*)\.log(")?\.$', stdout,
```
Since you don't need the capture group, use "? instead of (")?.  Then you don't have to change the group numbers below.

aclements commented on latexrun:
```diff
@@ -836,15 +836,15 @@ class LaTeX(Task):
             print(stdout, file=sys.stderr)
             raise TaskError('failed to extract job name from latex log')
-        for m in re.finditer(r'^Output written on (.*\.[^ .]+) \([0-9]+ page',
+        for m in re.finditer(r'^Output written on (")?(.*\.[^ ."]+)(")? \([0-9]+ page',
```
Same here.

aclements commented:
Interesting.  Do you know what triggers this?  It would be nice to add a test to the test suite if you do.

apuignav commented:
Thanks for the suggestion, you're absolutely right! I'll close this pull request and add another one where I have added the extra changes and squashed the commits to keep your tree clean.

As for when this happens, I am not completely sure, but I think it happens when the latex file name has spaces (at least in my version, MacTex  2014)

aclements commented:
BTW, I think if you just "git push -f" to update the branch that your pull request is on, GitHub will update the pull request.  Then all of the conversation and review history are in one place.
